tests/linearizability/linearizability_test.go

```diff
@@ -177,7 +179,7 @@ func TestLinearizability(t *testing.T) {
 				waitBetweenTriggers: waitBetweenFailpointTriggers,
 			}, *scenario.traffic)
 			forcestopCluster(clus)
-			validateWatchResponses(t, watchResponses)
+			validateWatchResponses(t, watchResponses, clus.Cfg.WatchProcessNotifyInterval != 0)
```

Comment:
Not sure I understand it correctly.
.
By default, the interval is 10 minutes. Currently, the test case won't hit the limit.
So `x != 0` can be used as disabled progress-notify? is that correct?
If so, is it possible to add some comment for this?

Reply:
> So x != 0 can be used as disabled progress-notify?

No, in e2e cluster config 0 is empty value which means, don't set. In that case etcd will use the default. As you mentioned the default interval is 10 minutes which is too much for test takes around 5 seconds.

Reply:
Thanks for the comment!